src/Concordium/Client/Output.hs

```diff
@@ -699,6 +727,23 @@ showEvent verbose = \case
     showDelegationTarget Types.DelegatePassive = "Passive delegation"
     showDelegationTarget (Types.DelegateToBaker bid) = "Baker ID " ++ show bid
 
+    showLoggedEvents :: [Wasm.ContractEvent] -> String
+    showLoggedEvents [] = "No contract events were emitted "
+    showLoggedEvents evs = [i|#{length evs} contract events were emitted|]
+      <> if isNothing stM
+         then [i| but no event schema was provided nor found in the contract module. |]
+         else [i|, of which #{length $ filter isJust $ map snd loggedEvents} were succesfully parsed using schema. |]
+      <> [i|Got:\n|]
+      <> intercalate "\n" [ [i|Event(raw): #{hex}\nEvent(JSON):\n#{indentBy 4 $ fromMaybe' json}|] | (hex, json) <- loggedEvents ]
```

Comment:
I don't think we need to print the hex value if we decode it successfully.
So I would suggest to just print one or the other.

Reply:
@abizjak what do you think?